Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/ICommentsManager.java

```diff
@@ -0,0 +1,24 @@
+package edu.asu.conceptpower.app.core;
+
+import edu.asu.conceptpower.core.CommentStatus;
+import edu.asu.conceptpower.core.ReviewRequest;
+
+public interface ICommentsManager {
+
+    /** Used to add comments for each concept entry
+     * @param conceptId - the conceptId of the concept word.
+     * @param comment - Comments provided by the user
+     * @param status - the status of the comment -open,resolved,etc.
+     */
+    public abstract void addComment(String conceptId,String comment ,CommentStatus status);
```

Comment:
you do need to know who is the requester though (sorry, rereading my previous comment, I realize it was a little ambiguous).